src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -1465,6 +1465,9 @@ void OrbitMainWindow::on_actionRename_Capture_File_triggered() {
       QFileDialog::getSaveFileName(this, "Rename or Move capture...",
                                    QString::fromStdString(current_file_path.string()), "*.orbit");
 
+  // Return if the rename operation is cancelled
+  if (file_path.isEmpty() || file_path.isNull()) return;
```

Comment:
Why do we need `isNull()` ?

Reply:
https://doc.qt.io/qt-5/qfiledialog.html#getOpenFileName

Ah, maybe `isNull` will be enough, we don't need `isEmpty`, let me double check

Reply:
Yes `isNull` is enough, I'm going to update the fix

Reply:
I think I would prefer `IsEmpty()` to be on the safe side. It checks both, whether its a null string or an empty string.

https://doc.qt.io/qt-6/qstring.html#isEmpty
https://doc.qt.io/qt-6/qstring.html#isNull